base/src/main/scala/co/uproot/abandon/BuildInfo.scala

```diff
@@ -0,0 +1,20 @@
+package co.uproot.abandon
+
+import scala.Predef._
+
+/** This object was generated by sbt-buildinfo. */
```

Comment:
It is. The problem is that I manually moved it from the main `target` folder to where it resides now.